projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-dashboard-widgets.php

```diff
@@ -45,7 +51,7 @@ function load_wpcom_dashboard_widgets() {
 			$wpcom_dashboard_widget['id'],
 			$wpcom_dashboard_widget['name'],
 			'render_wpcom_dashboard_widget',
-			function () {},
+			null,
```

Comment:
I'll have to adjust Phan, because this is totally allowed to be null https://developer.wordpress.org/reference/functions/wp_add_dashboard_widget/